### PR TITLE
Restyle pages mode with Gutenberg

### PR DESCRIPTION
**Pull request by amoshydra**
## Applying Gutenberg to Pages View
Gutenberg provide css styling that are more suitable for print medium.
Read more about [Gutenberg](https://github.com/BafS/Gutenberg/blob/master/README.md)

Comparison between Gutenberg theme with Default markdown

### CODE REVIEW

**Review by jiayingy — APPROVED** [review]

**Review by chuajiaxuan — COMMENTED** [review]
A possible addition would be the ability to let user print in color. As some users might want the code block to be color coded. This can be done in future development. :) Good job 👍

File: sashimi-webapp/static/styles/markdown-pages.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+@import url("/vendors/gutenberg-css/gutenberg.custom.css");
+@import url("/vendors/gutenberg-css/themes/oldstyle.min.css");
+@import url("/vendors/highlight.js/styles/ascetic.css");
+
+body {
+  text-align: initial;
+}
```
Comment (chuajiaxuan) [inline sashimi-webapp/static/styles/markdown-pages.css]: Is this ESLint warning??

**Review by chuajiaxuan — APPROVED** [review]
Nothing significant spotted! But do see the comment below. Good to go!

**Review by tharain — APPROVED** [review]
👍

File: sashimi-webapp/static/styles/markdown-pages.css (hunk 1 above)
Comment (amoshydra) [reply sashimi-webapp/static/styles/markdown-pages.css]: Fixed, added newline at the end of the file.
Btw.. I found this an answer and question about adding newline at the end of file. 
http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

**Comment by amoshydra** [comment]
@chuajiaxuan Noted your comment. Opened a new issue for that #373

**Review by tayjiehao — APPROVED** [review]

**Pull request merged**